### PR TITLE
feat: add touchbard as an option to control touch bar

### PR DESCRIPTION
**Pull request by NoaHimesaka1873**
Yes, it's complete enough to be suggested to users, I think.

### CODE REVIEW

**Comment by AdityaGarg8** [comment]
I'd like to know the difference between touchbard and the currently in use touchbar script.

**Comment by AdityaGarg8** [comment]
The difference in terms of functionality for the end user

**Comment by NoaHimesaka1873** [comment]
@AdityaGarg8 Changing touchbar mode does not change touchbar mode while booting. Currently touchbar script touches both module option and current touchbar mode.

**Comment by AdityaGarg8** [comment]
But I believe changing the mode while booting is more preferable. Atleast that what’s there in Windows Bootcamp.

**Comment by NoaHimesaka1873** [comment]
Bootcamp doing that doesn't mean we should follow its behavior. It gives user more choice since user can use particular mode one moment and upon reboot it will go back to default mode without manually changing back.

**Comment by AdityaGarg8** [comment]
Say for example I like having f keys by default always. So I’ll have to set them by in the terminal every time I boot Linux?

**Comment by NoaHimesaka1873** [comment]
No, there's config file to set boottime default.

**Comment by AdityaGarg8** [comment]
So users will have to manually edit the config file for the condition I meantioned previously.

**Comment by NoaHimesaka1873** [comment]
Yes. Although I have a plan to implement changing default mode via command line.

**Comment by AdityaGarg8** [comment]
> Yes. Although I have a plan to implement changing default mode via command line.

I believe we should have that implemented that first, cause I don't find a good reason to replace the touchbar script with touchbard.

Although @Redecorating and @kekrby may have separate views.

**Comment by NoaHimesaka1873** [comment]
That's why I placed it as secondary option!

**Comment by AdityaGarg8** [comment]
> That's why I placed it as secondary option!

Probably make it clear then.

File: docs/guides/postinstall.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ The steps to be followed vary depending upon the initramfs module loading mechan
 
 The Touch Bar can be set up by running [this script](../tools/touchbar.sh) **in Linux** using `bash /path/to/script`. Make sure your Linux kernel and macOS is updated before running this script.
 
+As a secondary option, if you want to try something new and you're using Arch Linux or EndeavourOS, you could install `touchbard` package and use that instead. For more information, visit [touchbard repository](https://github.com/NoaHimesaka1873/touchbard).
```
Comment (AdityaGarg8) [inline docs/guides/postinstall.md:94]: Isn't it available for Manjaro?

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/guides/postinstall.md (hunk 1 above)
Comment (NoaHimesaka1873) [reply docs/guides/postinstall.md:94]: I didn't upload the package to Manjaro repository. I shall upload and add Manjaro too.

**Pull request merged**